### PR TITLE
hotfix/scheduled-job-performance-stats

### PR DESCRIPTION
**Pull request by T-Sarwar**
### Change description ###
Add procedure required by dashboard performance stats job (scheduler execution layer) to flyway and migrate to juror_dashboard schema.

**Does this PR introduce a breaking change?** (check one with "x")

[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+language plpgsql as
+
+$$
+/*
+ * Populates juror_dashboard.stats_not_responded
+ * used by the Juror Digital performance dashboard
+ *
+ * Replaces the latest n months of summons counts as specified by the no_of_months input parameter
+ *
+ * It is recommended to use use no_of_months = 6
```
Comment (philhead1) [inline src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql:323]: Could we update the comment to remove the duplicated use please?

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+create or replace procedure juror_dashboard.response_times(no_of_months int)
+language plpgsql as
+
+$$
+/*
+ * Populates juror_dashboard.stats_response_times used by the Juror Digital performance dashboard
+ *
+ * Replaces the latest n months of summons months as specified by the no_of_months input parameter
+ *
+ * It is recommended to use use no_of_months = 6
```
Comment (philhead1) [inline src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql:426]: Could we update the comment to remove the duplicated use please?

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql [hunk 3]
```diff
@@ -0,0 +1,665 @@
...
+
+$$
+ /* Populates juror_dashboard.stats_not_responded and juror_dashboard.stats_response_times
+  * used by the Juror Digital performance dashboard
+  *
+  * Populating both tables in a single transacton so that they are always consistant with each other
+  *
+  * Replaces the latest n months of summons counts as specified by the no_of_months input parameter
+  *
+  * It is recommended to use use no_of_months = 6
```
Comment (philhead1) [inline src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql:108]: Could we update the comment to remove the duplicated use please?

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql (hunk 1 above)
Comment (T-Sarwar) [reply src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql]: Removed

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql (hunk 2 above)
Comment (T-Sarwar) [reply src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql]: Removed

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql (hunk 3 above)
Comment (T-Sarwar) [reply src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql]: Removed

**Review by philhead1 — APPROVED** [review]

File: src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql [hunk 4]
```diff
@@ -0,0 +1,646 @@
...
+       				date_trunc('month', coalesce(response_date, processed_date)),
+					case
+						when abs(coalesce(response_date::date, processed_date::date) - summons_date::date) < 8 then 'Within 7 days'
+			    		when abs(coalesce(response_date::date, processed_date::date) - summons_date::date) < 15 then 'Within 14 days'
+			    		when abs(coalesce(response_date::date, processed_date::date) - summons_date::date) < 22 then 'Within 21 days'
+			    		else 'Over 21 days'
+			    	 end,
+       				s.loc_code, s."method"
+    ON CONFLICT(summons_month, response_month, response_period, loc_code, response_method)
+        DO update set response_count = m.response_count;
```
Comment (philhead1) [inline src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql:493]: I believe the m.response_count should be s.response_count

**Review by philhead1 — APPROVED** [review]

**Pull request merged**